spec/support/analyzer_spec.rb

```diff
@@ -0,0 +1,28 @@
+module AnalyzerSpec
+  def self.included(base)
```

Comment:
Would this be better added via `RSpec.configure`? Maybe on a `in_tmpdir: true` tag or something?